dash_bio/component_factory/_clustergram.py

```diff
@@ -50,15 +50,14 @@ def Clustergram(
 Keyword arguments:
 
 - data (ndarray; required): Matrix of observations as array of arrays
-- generate_curves_dict (bool; optional): Whether or not to return a
+- generate_curves_dict (bool; default false): Whether or not to return a
     dictionary containing information about the cluster number
     associated with each curve number in the graph. (May be useful
     if one wishes to capture the cluster number that is clicked.)
```

Comment:
```suggestion
    for capturing the cluster number that is clicked.)
```
I know it's not part of the PR, just noticed while reviewing...